sources/validation-api.asciidoc

```diff
@@ -912,76 +912,63 @@ package com.acme.constraint;
 include::{spec-examples-source-dir}org/beanvalidation/specexamples/validationapi/NonEmpty.java[tags=include]
 ----
 
-and the following class definitions
+And the following class definitions:
 
 [source, JAVA, indent=0]
 ----
 include::{spec-examples-source-dir}org/beanvalidation/specexamples/validationapi/Author.java[tags=include]
+----
 
+[source, JAVA, indent=0]
+----
 include::{spec-examples-source-dir}org/beanvalidation/specexamples/validationapi/Book.java[tags=include]
+----
 
-Author author = new Author();
-author.setCompany("ACME");
-Book book = new Book();
-book.setTitle("");
-book.setAuthor(author);
+When executing the following validation:
 
-Set<ConstraintViolation> constraintViolations = validator.validate(book);
+[source, JAVA, indent=0]
+----
+include::{spec-examples-source-dir}org/beanvalidation/specexamples/validationapi/ValidationApiTest.java[tags=invocation]
 ----
 
-[classname]`ConstraintViolations` is a set of size 2. One of the entries represents the failure of `@NonEmpty` (or more precisely [classname]`@Size(min=1)` a composing constraint of [classname]`@NonEmpty`) on the `title` property.
+Then `constraintViolations` is a set of size 3. One of the entries represents the failure of `@NonEmpty` (or more precisely [classname]`@Size(min=1)` a composing constraint of [classname]`@NonEmpty`) on the `title` property.
 
 The [classname]`ConstraintViolation` object for this failure passes the following assertions:
 
-.Test assertions on ContraintViolation
+.Test assertions on `ContraintViolation` for `title`
 ====
 
-[source, JAVA]
+[source, JAVA, indent=0]
 ----
-//assuming an english locale, the interpolated message is returned
```

Comment:
These sections don't have changed at all, I just pulled the into actual source files to be sure they compile and are correct.